### PR TITLE
Added Species & Owner Table, Ran Given Queries

### PR DESCRIPTION
**Pull request by Shakir-Hussain12**
**For this PR, I :**
- Created `Owners` Table.
- Created `Species` Table.
- Modified `Animals` table to exclude species column.
- Added `species_id` and `owner_id` to `Animals` table.
- Added data to the new tables and columns.
- Used `JOIN` to run the required queries, results of which are attached below...
---
q1
q2
q3
q4
q5
q6
q7

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,14 +1,73 @@
...
+drop column species;
+commit;
+
+begin;
+alter table animals
+add column species_id int references species(id);
+
+alter table animals
+add column owner_id int references owners(id);
+commit;
```
Comment (TadesseDev) [inline data.sql]: - [x] Dear @Shakir-Hussain12, Please remember everything that creates or `changes` the structure of the table should be saved in the `schema.sql` file. as a result of this these `ALTER` statements should be written inside your `schema.sql` file.

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @Shakir-Hussain12 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7

## Highlights
 - [x]  Correct GitFlow  ✅
 - [x]  Good PR title/description  ✅
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [Optional] Please try to separate your code into different levels of commits. Remember your commits are going to help you look back and debug your code. It's a good idea to commit after every visible change.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (Shakir-Hussain12) [reply data.sql]: Sorry @TadesseDev , It took a while mainly because of some electricity issues.

Comment (TadesseDev) [reply data.sql]: That's all right @Shakir-Hussain12 👏

**Review by KayLemba — APPROVED** [review]
[[# STATUS: APPROVED :green_circle: :clap: 

Hello @Shakir-Hussain12    , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉  Keep up the good work 🥂 

HIGHLIGHTS:

- [x]  Your repo is professional  😎 
- [x] Descriptive P.R 💯
- [x] Professional README 💯
- [x] Your work is well documented 🥇 
- [x] Project setup correctly 🥇  











 








Now you can merge this branch and move on :1st_place_medal: 
Happy Coding cheers 🥇 💯 



well-1

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._]]s

**Pull request merged**